src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ValueTask.cs

```diff
@@ -398,6 +398,15 @@ internal void ThrowIfCompletedUnsuccessfully()
             }
         }
 
+        /// <summary>Transfers the <see cref="ValueTask{TResult}"/> to a <see cref="ValueTask"/> instance.</summary>
+        /// <remarks>The <see cref="ValueTask{TResult}"/> should not be used after calling this method.</remarks>
```

Comment:
This is an internal method. I'm ok with having triple slash comments, but putting that text in the remarks means it won't show up in intellisense. And since it's not ported to MS Docs (because it's internal), I suggest we put all the text in the summary.

```suggestion
        /// <summary>Transfers the <see cref="ValueTask{TResult}"/> to a <see cref="ValueTask"/> instance.
        /// The <see cref="ValueTask{TResult}"/> should not be used after calling this method.</summary>
```